client_api.go

```diff
@@ -426,6 +451,41 @@ func (c *Client) ClaimHTTP(ctx context.Context, id, authDomain, scheme string) (
 	return c.claimAssert(ctx, body, id, pathHTTP)
 }
 
+// ClaimEmail requests assertion of domain control using Email once the appropriate
```

Comment:
This isn't how email domain claims work.  Posting to /email with an email address in the body requests that a link be sent to that email address which will be followed by the user in order to prove ownership of the domain

Reply:
Yes I think I just neglected the comment clean up, good spot